types/chart.js/chart.js-tests.ts

```diff
@@ -100,6 +100,21 @@ const chart: Chart = new Chart(ctx, {
                 },
             ],
         },
+        elements: {
+            rectangle: {
+                backgroundColor(ctx) {
+                    if (typeof ctx.dataset?.backgroundColor === "function") {
```

Comment:
I think the optional chaining here is what triggered the typescript compilation error, and this is the only thing that looks like it requires [typescript 3.7](https://en.wikipedia.org/wiki/TypeScript#Release_history:~:text=Optional%20Chaining).

It would be better to remove the optional chaining in the tests, and  leave the original TS versions

Reply:
Yep: https://github.com/DefinitelyTyped/DefinitelyTyped/runs/1929293682

> ERROR: 114:65  expect  Compile error in typescript@3.6 but not in typescript@3.7.
Fix with a comment '// Minimum TypeScript Version: 3.7' just under the header.
':' expected.

Reply:
Ah thanks will do